v3/src/components/case-table/case-table-model.ts

```diff
@@ -37,13 +43,22 @@ export const CaseTableModel = TileContentModel
     }
   })
   .actions(self => ({
+    setColumnWidth(attrId: string, width?: number) {
+      if (width) {
+        self.columnWidths.set(attrId, width)
+      }
+      else {
+        self.columnWidths.delete(attrId)
+      }
+    },
     updateAfterSharedModelChanges(sharedModel?: ISharedModel) {
       // TODO
     },
     setScrollLeft(scrollLeft: number) {
       self.scrollLeft = scrollLeft
     }
   }))
+  .actions(applyUndoableAction)
```

Comment:
This should be added to the `TileContentModel` (I'm surprised it hasn't already been added there) so that it's present for all tiles.
```suggestion
```